### ISSUE TITLE
Remove wrapped div

### ISSUE THREAD

**Issue by mthines** [developit/preact-markup#19]
I'm creating a translation module which needs to render html from a string.
I want inline text to have different translate strings, so I'm creating a component which can wrap those strings. 
Is it possible to make it not wrap the markup in a div?

### My code (JSX)
```jsx
import Markup from 'preact-markup';

<h6>
  <Markup markup={'The <strong>price</strong> is'}/>
  <span className="price">680</span>
  <Markup markup={'$'}/>
</h6>
```

### What I expect to be outputted
```jsx
<h6>
  The <strong>price</strong> is
  <span className="price">680</span>
  $
</h6>
```

### What's being outputted
```jsx
<h6>
  <div class="markup">The <strong>price</strong> is</div>
  <span className="price">680</span>
  <div class="markup">$</div>
</h6>
```

How can I remove the div? :)

Thanks! 💪

**Comment by mads-thines** [comment]
@elias6 As far as my needs goes, the `dangerouslySetInnerHTML` solves my issue, so that's why I closed the issue :)
But I will reopen, to keep the question alive :)

**Comment by matthijsgroen** [comment]
I think this is possible using:

```javascript
const wrapSpan = text => `<span>${text}</span>`;
```

```javascript
<h6>
  <Markup wrap={false} markup={wrapSpan('The <strong>price</strong> is')}/>
  <span className="price">680</span>
  <Markup wrap={false} markup={wrapSpan('$')}/>
</h6>
```

**Comment by developit** [comment]
The wrapping div can now be removed in Preact X. I'll add an option for this.

**Comment by donovantc** [comment]
@developit did you ever get around to removing the wrapping div? I could not find the change in v2.0.0 the code and I just want to be sure I'm not missing something?

**Comment by developit** [comment]
This shipped in [2.1.0](https://github.com/developit/preact-markup/releases/tag/2.1.0).
